### PR TITLE
Db character set

### PR DESCRIPTION
**Pull request by mgarciaberlin**
To see the default character set and collation for a given database:

```
USE db_name;
SELECT @@character_set_database, @@collation_database;
```

### CODE REVIEW

File: core/src/main/resources/db/migration/V1__initial_schema.sql [hunk 1]
```diff
@@ -1,3 +1,5 @@
+ALTER DATABASE CHARACTER SET = utf8;
+
```
Comment (jerrytrieu) [inline core/src/main/resources/db/migration/V1__initial_schema.sql]: Let's not change this V1 sql.

Comment (mgarciaberlin) [inline core/src/main/resources/db/migration/V1__initial_schema.sql]: Alternatives:
1. add to the last migration step the `ALTER DATABASE CHARACTER SET = utf8;`
2. add a new migration step consisting solely of the `ALTER DATABASE` above.

Comment (jerrytrieu) [inline core/src/main/resources/db/migration/V1__initial_schema.sql]: V1 and V2 are already in JLR repo. Let's change in V3.

**Comment by mgarciaberlin** [comment]
The character set for the database is now set in the V3 migration steps of core and resolver, as can be seen in https://github.com/advancedtelematic/rvi_sota_server/pull/350/commits/00cbb2205fe6942d0d65a1cc1aa8ec3c0ba1edde

**Pull request merged**